bench/common.ml

```diff
@@ -0,0 +1,76 @@
+let reporter ?(prefix = "") () =
```

Comment:
don't forget the copyright headers when you create a new file :-)

Reply:
there are no copyright headers in any file in the index

Reply:
good point! happy to have a fix in a separate PR